ros_speech_recognition/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package>
+  <name>ros_speech_recognition</name>
+  <version>0.0.0</version>
+  <description>Speech Recognition</description>
```

Comment:
please describe more detail about this package, ros interface for python-speechrecognition module etc...